uwsgi/run.sh

```diff
@@ -6,6 +6,12 @@ elif [ $1 == "admin" ]; then
    exec uwsgi --ini /app/uwsgi/admin.ini --python-autoreload 1
 elif [ $1 == "admin-dev" ]; then
    exec uwsgi --ini /app/uwsgi/admin.dev.ini --ini /app/uwsgi/admin.ini --python-autoreload 1
+elif [ $1 == "upgrade-db" ]; then
+    if [ -z ${DBURI} ]; then
```

Comment:
I think the variable should be quoted, I trust our old scripts :) http://hg.mozilla.org/build/tools/file/tip/release/updates/verify.sh#l148